Resources/Prototypes/Catalog/Fills/Lockers/misc.yml

```diff
@@ -68,14 +68,58 @@
   components:
   - type: StorageFill
     contents:
+      - id: ClothingOuterSuitEmergency
       - id: ClothingMaskBreath
       - id: EmergencyNitrogenTankFilled
         prob: 0.80
         orGroup: EmergencyTankOrRegularTank
       - id: NitrogenTankFilled
         prob: 0.20
         orGroup: EmergencyTankOrRegularTank
+      - id: ToolboxEmergencyFilled
+        prob: 0.4
+      - id: WeaponFlareGun
+        prob: 0.05
+      - id: BoxMRE
+        prob: 0.1
 
+- type: entity
+  id: ClosetImprovisedEmergencyN2FilledRandom
+  parent: ClosetImprovisedEmergencyN2
+  suffix: Filled, Random
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingMaskBreath
+      - id: EmergencyNitrogenTankFilled
+        prob: 0.80
+        orGroup: EmergencyTankOrRegularTank
+      - id: NitrogenTankFilled
+        prob: 0.20
+        orGroup: EmergencyTankOrRegularTank
+        
+- type: entity
+  id: ClosetWallEmergencyN2FilledRandom
+  parent: ClosetWallEmergencyN2
+  suffix: Filled, Random
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingOuterSuitEmergency
+      - id: ClothingMaskBreath
+      - id: EmergencyNitrogenTankFilled
+        prob: 0.80
+        orGroup: EmergencyTankOrRegularTank
+      - id: NitrogenTankFilled
+        prob: 0.20
+        orGroup: EmergencyTankOrRegularTank
+      - id: ToolboxEmergencyFilled
+        prob: 0.4
+      - id: WeaponFlareGun
+        prob: 0.05
+      - id: BoxMRE
+        prob: 0.1
+ 
```

Comment:
Please don't copy-paste yaml, just add parenting to storagefill if you want it instead of copying yaml over and over.

Reply:
Do you mean this is as an existing feature I should use or as a feature I should implement? If it's the former, I'd appreciate an example on how it's used. If it's the later, it's a bit beyond the scope of this PR and my skills.